homeassistant/components/mqtt/climate.py

```diff
@@ -668,84 +715,10 @@ async def _subscribe_topics(self):
         """(Re)Subscribe to topics."""
         await subscription.async_subscribe_topics(self.hass, self._sub_state)
 
-    @property
-    def temperature_unit(self) -> str:
-        """Return the unit of measurement."""
-        if unit := self._config.get(CONF_TEMPERATURE_UNIT):
-            return unit
-        return self.hass.config.units.temperature_unit
-
     @property
     def current_temperature(self) -> float | None:
         """Return the current temperature."""
```

Comment:
Is there a reason to keep this property? Or was it just missed in the refactoring?